services/app-web/src/pages/StateSubmission/RateDetails/RateDetailsSchema.ts

```diff
@@ -10,166 +10,177 @@ import {
 Yup.addMethod(Yup.date, 'validateDateFormat', validateDateFormat)
 
 const SingleRateCertSchema = (_activeFeatureFlags: FeatureFlagSettings) =>
```

Comment:
I changed this schema by 1. removing rate edit unlock feature flag logic from linked rates fields since seems like now this logic just hinges on linked rates 2. making it a three part validation instead of two part validation to err on side of being more explicit with the logic tree